src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -204,6 +204,7 @@ public void Version_Valid()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // throws PNSE
```

Comment:
@steveisok this will also be reenabled as part of your PR right?

Reply:
Yes.  Probably makes sense to just exclude this b/c I modified the test.